### DISCUSSION TITLE
How do I transfer my license to a new computer?

### DISCUSSION THREAD

**Discussion by kentzh1** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#542] [Q&A] [answered]
There are three devices registered for the AIdente pro license. My computer needs to be replaced due to quality problems. If I choose to deactivate the license, will it affect the registration of other devices? Can I deactivate the license, uninstall the software, and then install and register the license directly after the new computer arrives?

**Comment by MatthiasKerbl** [comment, marked as answer]
> There are three devices registered for the AIdente pro license. My computer needs to be replaced due to quality problems. If I choose to deactivate the license, will it affect the registration of other devices? Can I deactivate the license, uninstall the software, and then install and register the license directly after the new computer arrives?

Hi @kentzh1,

That's correct. Just deactivate the license on your old MacBook and activate it on your new machine again. The other MacBooks should not get affected.

Cheers,
Matthias
